package/src/components/MessageInput/components/InputReplyStateHeader.tsx

```diff
@@ -15,25 +16,22 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     flexDirection: 'row',
     justifyContent: 'space-between',
-    paddingBottom: 10,
+    paddingBottom: 8,
```

Comment:
Again, why are we introducing styling changes on a whim like this ? They are all pretty much breaking and unless we have a very good reason for this we probably should not do them.

Reply:
Here the padding was a bit more and looked uneven that is why I reduced it. Shouldn't be that big of a breaking change. I can remove it if you feel otherwise.

Reply:
I don't mind it if it's fixing UI, just pointing it out. Downstream integrations may be doing stuff differently because of this. Let's have the change for now and see how that goes.